src/radiation/monte_carlo.cpp

```diff
@@ -514,7 +623,7 @@ TaskStatus MonteCarloUpdateTuning(Mesh *pmesh, std::vector<Real> *resolution,
 
     // TODO(BRR) This should be Rout_rad (add it) or max cartesian size, and also actually
     // used.
-    // const Real L = 1.;
+    const Real L = std::exp(pmesh->mesh_size.x1max);
```

Comment:
We're going to need to update this.. It's supposed to be e.g., `Rout_rad` and this is a hack